resources/views/auth/register.html.twig

```diff
@@ -151,11 +150,9 @@
                                         type="password"
                                         autocomplete="off" />
                                 </div>
-
                             </div>
-
                         </div>
-
+                        <span class="password-caps-warning hidden">Warning: Caps lock engaged</span>
```

Comment:
My password manager adds a div underneath the password field and obscures the text. Would above be better for most cases? Not a big deal.